packages/menu/src/createMenuItem.ts

```diff
@@ -106,9 +106,9 @@ export function createMenuItem<T extends HTMLElement>(
 ): MenuItemAria {
   const context = useMenuContext();
 
-  const labelId = createSlotId();
-  const descriptionId = createSlotId();
-  const keyboardId = createSlotId();
+  const [labelId] = createSlotId();
```

Comment:
@thetarnav why don't you call the track function in `labelProps`, `descriptionProps` and `keyboardShortcutProps` below in this case ? (same in `createDialog`)

Reply:
If it didn't use the deps array before, then I didn't use track. Don't know when it's necessary to track, and where it's not.
But I guess you should always since this is what the primitive is for.

Reply:
We need to track when the element that the id is spread on may appear/disappear due to some signal/props changes, for example an input with an error message.
When there is no deps array we should always perform one track to check if the generated id is used or not.